index.html

```diff
@@ -324,12 +324,24 @@
                         let printText = document.getElementById("printText");
                         printText.classList.remove("show");
                     }
-
+                
```

Comment:
why are these spaces here?

Reply:
Sir i have removed the extra spaces. please check